api/managementpb/dbaas/pxc_clusters.proto

```diff
@@ -135,6 +143,10 @@ message UpdatePXCClusterRequest {
       // Image to use. If it's the same image but with different version tag, upgrade of database cluster to version
       // in given tag is triggered. If entirely different image is given, error is returned.
       string image = 2;
+      // Configuration for PXC cluster
+      string configuration = 4;
+      // Storage Class for PXC cluster.
+      string storage_class = 5;
```

Comment:
Given that there isn't a field with index 3 shouldn't these be 3 and 4, respectively? Did you purposefully did this to be more consistent with `PXCClusterParams` above?

Reply:
image field has index #3